roles/jito/tasks/build.yaml

```diff
@@ -1,7 +1,7 @@
 - name: build solana
   become: true
   become_user: root
-  shell: "git config --global --add safe.directory /mnt/{{ repo_dir }} && cd /mnt/{{ repo_dir }} && ./f"
+  shell: "source /home/solana/.cargo/env && /mnt/{{ repo_dir }}/cargo stable build --release"
```

Comment:
does this work? i think you need to cd there?

Reply:
also should we run cargo-install-all like this? ideally the install location somewhat similar to solana-install


https://jito-foundation.gitbook.io/mev/jito-solana/building-the-software#initial-setup

Reply:
also if you run cargo install all, should prob be under solana user so it gets installed in correct location